tests/ui/type-alias-impl-trait/issue-109054.stderr

```diff
@@ -0,0 +1,18 @@
+error: expected one of `!`, `.`, `::`, `;`, `?`, `{`, `}`, or an operator, found keyword `move`
```

Comment:
this test isn't correct :)

needs edition 2021

Reply:
Good call, fixed!